NEWS.md

```diff
@@ -1,5 +1,6 @@
 # dbplyr 1.1.0.9000
 
+* `names_to_as()` now prevents select from adding names when not necessary (#2968) (@DavisVaughan)
```

Comment:
How about:

```
* Queries generated by `select()` no longer alias column names 
  unnecessarily (#2968) (@DavisVaughan).
```

Would you like to have an example to explain further?

```
* Queries generated by `select()` no longer alias column names unnecessarily:
  ``SELECT `col_name` ``, not ``SELECT `col_name` AS `col_name` `` 
  (#2968) (@DavisVaughan).
```

This renders correctly when I preview NEWS, but looks kind of ugly because of the double backticks.